vmdb/lib/workers/worker_base.rb

```diff
@@ -421,7 +421,8 @@ def parent_is_alive?(parent)
   end
 
   def check_parent_process
-    server = MiqServer.my_server(true)
+    # ensure attributes like "last_heartbeat" are up to date
+    server.reload if server
```

Comment:
@blomquisg please delete both the methods, see information below...

`WorkerBase#check_parent_process` was last used for sql based worker heartbeating and wasn't removed when we deleted sql in favor of drb based heartbeats here:

```
1f3dc7595e8a9a0adfd528af69cba5af1d900f86
Date:   Tue Jan 24 17:10:07 2012 +0000

    Remove SQL-based Worker heartbeating so that DRb Heartbeat is the only mode of heartbeating
    - Remove message and monitor_status columns from miq_workers table
    - Worker Stoppage is now routed exclusively via MiqServer#stop_worker method
    - Clean up termination of UI and WebService Workers (the ones wrapping a Web Server, i.e. Mongrel)
    - Removed unused shutdown method from all workers
    - Added MiqProcess.alive? method
    - Use MiqWorker::STATUS_ constants, where appropriate
    - Removed no longer used heartbeat_method from vmdb.tmpl.yml
```

`WorkerBase#parent_is_alive?` was last used by the WorkerMonitor circa 2010 until this commit deleted the last caller of it:

```
8b8ab8d64a609b21024eab679f3890cb90fb3f19
Date:   Mon Jul 12 18:59:31 2010 +0000

    Added do_system_limit_exceeded method to MiqServer
    Removed app/helpers/workers/worker_monitor.rb and app/models/miq_worker_monitor.rb
```